CafeTime/CafeTime/StringsChecker.swift

```diff
@@ -32,7 +32,7 @@ class StringsChecker {
     // sign up details
```

Comment:
we can call this one Validator or something similar. don't like "StringsChecker" name, cause in future in can me extended not only yo check strings, but for password validation for example